Docs/source/install/cmake.rst

```diff
@@ -173,29 +176,87 @@ Run
+
+.. code-block:: bash
+
+   # find dependencies & configure
+   cmake -S . -B build -DWarpX_LIB=ON
+
+   # build and then call "python3 -m pip install ..."
+   cmake --build build --target pip_install -j 4
+
+**That's it!**
+You can now :ref:`run a first 3D PICMI script <usage-picmi>` from our :ref:`examples <usage-examples>`.
+
+Developers could now change the WarpX source code and then call the build line again to refresh the Python installation.
+
+.. note::
+
+   These commands builds one ``-DWarpX_DIMS=...`` dimensionality (default: ``3``) at a time.
+   If your ``build/lib*/`` directory contains previously built ``libwarpx*`` libraries, then ``--target pip_install`` picks them up as well.
+   A new call to ``cmake --build build ...`` will only rebuild *one* dimensionality, as set via ``WarpX_DIMS``.
+
+If you like to build a WarpX Python package that supports *all* dimensionalities, you can run this:
+
+.. code-block:: bash
+
+   for d in 1 2 3 RZ; do
+       cmake -S . -B build -DWarpX_DIMS=$d -DWarpX_LIB=ON
+       cmake --build build -j 4
+   done
+   cmake --build build --target pip_install
+
+.. tip::
+
+   If you do *not* develop with :ref:`a user-level package manager <install-dependencies>`, e.g., because you rely on a HPC system's environment modules, then consider to set up a virtual environment via `Python venv <https://docs.python.org/3/library/venv.html>`__.
+   Otherwise, without a virtual environment, you likely need to add the CMake option ``-DPYINSTALLOPTIONS="--user"``.
+
+
+.. _building-pip-python:
+
+Python Bindings (Package Management)
+------------------------------------
+
+This section is relevant for Python package management, mainly for maintainers or people that rather like to interact only with ``pip``.
+
+One can build and install ``pywarpx`` from the root of the WarpX source tree:
 
 .. code-block:: bash
 
    python3 -m pip wheel -v .
-   python3 -m pip install *whl
+   python3 -m pip install pywarpx*whl
+
+This will call the CMake logic above implicitly.
+Using this workflow has the advantage that it can build and package up multiple libraries with varying ``WarpX_DIMS`` into one ``pywarpx`` package.
 
 Environment variables can be used to control the build step:
 
 ============================= ============================================ ================================================================
 Environment Variable          Default & Values                             Description
 ============================= ============================================ ================================================================
 ``WARPX_COMPUTE``             NOACC/**OMP**/CUDA/SYCL/HIP                  On-node, accelerated computing backend
-``WARPX_DIMS``                ``"2;3;RZ"``                                 Simulation dimensionalities (semicolon-separated list)
+``WARPX_DIMS``                ``"1;2;3;RZ"``                               Simulation dimensionalities (semicolon-separated list)
```

Comment:
In the other table above we write simply 3/2/1/RZ, without the extra formatting `"..."`. Shall we make them uniform (meaning either both without formatting like the one above or both with formatting like the one here -  I personally find the one without formatting easier to read from within the table in this case)?

Reply:
Ah, but the one here is a literal default that contains `;`s.